packages/lesswrong/server/callbacks/rateLimits.ts

```diff
@@ -82,6 +251,121 @@ async function enforcePostRateLimit (user: DbUser) {
   }
 }
 
+async function enforceCommentRateLimit({user, comment}:{user: DbUser, comment: DbComment}) {
```

Comment:
this is just moved up from elsewhere in the file to group it with enforcePostRateLimit